packages/string-templates/src/processors/preprocessor.js

```diff
@@ -71,6 +71,9 @@ module.exports.processors = [
     ) {
       insideStatement = `(${insideStatement})`
     }
+    if (statement === "{{ uuid }}") {
```

Comment:
Maybe this should be at the top of the function to better show intent, as the `insideStatement` does not need to be calculated for this path.

Also are we sure that this statement couldn't be `{{uuid}}` for example?

Reply:
sorry this wasn't needed - was introduced during testing, removed!